### PR TITLE
fix core esm, types and style  exports

### PR DESCRIPTION
**Pull request by jaketrent**
- Bringing the core package in line with other package builds.
- Fixing the module export -- now it's a real esm build
- Fixing the style export -- now it's the full css module instead of the utility stylesheet
- Fixing the types export -- now it actually exists.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-11.4.0
+15.9.0
```
Comment (danethurber) [inline .nvmrc:1]: make sure to look for versions in all the places: package.json>engine, gitlab runners, dockerfiles, docs dir

**Review by danethurber — APPROVED** [review]
pulled and rebuilt everything. ran the tests and they passed. LGTM 👍

**Review by EdwardIrby — APPROVED** [review]

**Comment by jaketrent** [comment]
@danethurber, did I update all the right node versions? 39efdf9

The one I wasn't sure about was gitlab.  I don't see anything in the private/.gitlab-ci.yml that would need updating.

**Comment by danethurber** [comment]
@jaketrent i think you got them all

**Pull request merged**